awsiot/mqtt_connection_builder.py

```diff
@@ -482,10 +487,10 @@ def direct_with_custom_authorizer(
     else:
         username_string += auth_username
 
-    if not auth_authorizer_name is None:
+    if auth_authorizer_name is not None:
```

Comment:
Unrelated to this PR but: We probably want to change this to `!= None` instead of `is not None` because we got a GitHub issue where someone was having Python warnings in their logs using Python 3.8+ due to the use of `is`.

Reply:
any use of '!= None' is being auto-formatted to 'is not None' for me. We should probably tackle this as a separate issue and fix it everywhere if we fix it anywhere =]